pkg/metrics/granularmetric.go

```diff
@@ -0,0 +1,226 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package metrics
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/cilium/tetragon/pkg/metrics/consts"
+	"github.com/cilium/tetragon/pkg/option"
+	"github.com/prometheus/client_golang/prometheus"
+	"golang.org/x/exp/slices"
```

Comment:
hey @lambdanis I think you reintroduced `golang.org/x/exp/slices` here, I don't know if it's intended since you removed that https://github.com/cilium/tetragon/pull/1560.

Reply:
I just rebased one of my PR and noticed that go.mod needed to include the new dependency.

Reply:
I think the tests passed on this PR a while ago, before some deps got updated?
Anyway, thanks for fixing this import.